src/js/smooch.jsx

```diff
@@ -196,6 +195,40 @@ export class Smooch {
         });
     }
 
+    monitorUrlChanges() {
+        const onhashchange = window.onhashchange;
+        const pushState = window.history.pushState;
+        const replaceState = window.history.replaceState;
+
+        window.onhashchange = function() {
```

Comment:
actually these should probably use this format : `window.addEventListener("hashchange", fn, false);`

That's the way it's done for other global events in this project and I'd prefer we stick we the same convention.